### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by craastad**

### CODE REVIEW

File: content/notes/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,24 @@ aliases:
 weight: 1
 ---
 
+## May 2021
+
+### Conditional build status and conditional build steps
```
[the remark this answers is not in the document]
Comment (craastad) [reply content/notes/release-notes.md]: Good catch  👀

File: content/notes/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,24 @@ aliases:
...
+
+### Flutter macOS and Linux desktop publishing
+
+You can now build and publish Flutter apps to the macOS App Store. This can be configured in [codemagic.yaml](../publishing-yaml/distribution/#app-store-connect) or [Flutter workflow editor](../publishing/publishing-to-app-store/). See the step-by-step macOS code signing guide for [codemagic.yaml](../code-signing-yaml/signing-macos/) and [Flutter workflow editor](../code-signing/macos-code-signing/). macOS publishing is limited to [macOS build machines](../specs/machine-type/#mac-mini-and-mac-pro).
+
+You can now build and publish Flutter apps to the Linux [Snapcraft Snap Store](https://snapcraft.io/snap-store). This can be configured in [codemagic.yaml](../getting-started/building-a-snap-package/) or [Flutter workflow editor](../publishing/snap-store/). Linux publishing is limited to [Linux build machines](../specs/machine-type/#linux-standard-and-linux-premium).
```
Comment (helinanever) [inline content/notes/release-notes.md:19]: Actually, when using yaml, it does not need to be a flutter-based app.

Comment (helinanever) [reply content/notes/release-notes.md:19]: And I think the same applies to macOS. however, considering our user base, I don't think it's necessary to go into too many details here

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**